docs/specs/openapi.json

```diff
@@ -7981,6 +8051,56 @@
                     }
                 }
             },
+            "ReportAWSEC2ComputeFilter": {
```

Comment:
This section is missing `operating_system`

Reply:
Oof, nice catch. I've added it.